cpp/harvest/core_issn_lookup.cc

```diff
@@ -0,0 +1,51 @@
+/**
+ * \brief The Utility for extracting issn information from https://portal.issn.org/
+ * \author Steven Lolong (steven.lolong@uni-tuebingen.de)
+ *
+ * \copyright 2023 Tübingen University Library.  All rights reserved.
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <iostream>
+#include "IssnLookup.h"
+#include "util.h"
+
+
+[[noreturn]] void Usage() {
+    ::Usage(
+        "issn [--verbose]\n"
```

Comment:
It's good that you renamed the parameter to `--verbose`, since this similar to many of our other programs.
For even bigger similarity, you could move the `--verbose` parameter to the beginning.